Sources/App/Environments/EditorEnvironment.swift

```diff
@@ -0,0 +1,11 @@
+//
+//  Mocka
+//
+
+import Foundation
+
+/// An `Environment` object shared by all the `View`s of the editor section.
+final class EditorEnvironment: ObservableObject {
```

Comment:
Should we rename it to `EditorSectionEnvironment`, because is used inside the `EditorSection`. By using `EditorEnvironment` it can be confused with the `Editor` environment (that we don't have)

```suggestion
final class EditorSectionEnvironment: ObservableObject {
```